spring-boot/admin/pom.xml

```diff
@@ -37,6 +37,7 @@
         <dependency>
             <groupId>org.springframework.session</groupId>
             <artifactId>spring-session</artifactId>
+            <version>1.3.5.RELEASE</version>
```

Comment:
Whenever I added a specific version number, I did it because there was an error message during the building process that stated to explicitly do so. I'm afraid that I can't give you an explanation for that, but we should create an issue about that to investigate this strange behavior. Would you agree?

Reply:
I did a quick check of the migration guide, and it links to the following overview of Spring dependency management, comparing v1.5 and v2.0:

* https://docs.spring.io/spring-boot/docs/1.5.x/reference/html/appendix-dependency-versions.html
* https://docs.spring.io/spring-boot/docs/2.0.x/reference/html/appendix-dependency-versions.html

When searching for `spring-session`, one can see that Spring Boot 2.0 does not include it as dependency anymore, but instead it includes `spring-session-core`. So probably this dependency should be replaced by `spring-session-core` version `2.0.10.RELEASE`. However, due to possible API changes that could be additional work so I would suggest a follow-up issue to cope with situations like this to not further delay the merge.

Probably it would make sense to create a general follow-up issue to re-check the explicitly set version numbers, which probably are related to outdated libraries, and replace them with the appropriate Spring 2.0 dependency.

Reply:
The funny thing is, that I learned that we don't need to declare neither dependency manually. `spring-session` is not used and `spring-session-core` is implicitly provided due to another package. I could see that by generating the effective pom.xml.

Reply:
I created the follow-up issue #185 about this matter. Feel free to add your thoughts there. This thread can be closed imo.